images/win/scripts/Installers/Install-NodeLts.ps1

```diff
@@ -11,7 +11,15 @@ New-Item -Path $PrefixPath -Force -ItemType Directory
 New-Item -Path $CachePath -Force -ItemType Directory
 
 $defaultVersion = (Get-ToolsetContent).node.default
-Choco-Install -PackageName nodejs -ArgumentList "--version=$defaultVersion"
+$queryFilter = "`$filter=(Id eq 'nodejs') and (IsPrerelease eq false) and (Version ge '$defaultVersion')"
```

Comment:
Would it make sense to extend this to include an upper limit?

```
`$filter=(Id eq 'nodejs') and (IsPrerelease eq false) and (Version ge '$defaultVersion') and (Version lt '15.0')
```

Reply:
This solution will differ from our regular approach which we use for versions in toolset file and probably can confuse someone.